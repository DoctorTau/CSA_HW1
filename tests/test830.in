575
-7838 3865 3601 1584 -2856 -5637 1988 3258 2240 -8753 5894 4738 -3431 8906 -7054 6543 -5777 -2638 4938 -8564 -6858 -4899 -7740 -5905 7301 4968 3499 -1295 -4696 7202 -8662 -9948 6885 7845 6665 -411 8450 4788 7140 6047 -5564 -6722 -5184 6611 2817 -6590 -8826 7283 3885 -167 7277 -6532 1811 -4433 8014 -9811 9036 8685 7535 7203 -3307 5708 9078 2861 9358 7845 -6748 -2209 9721 -1414 4219 944 -1694 -941 2046 -2664 -2216 8872 660 7378 -8078 3684 -6580 3993 1648 9301 -211 6607 7528 1298 6731 2766 -669 -5207 -3530 1635 1782 -2384 5276 -770 4348 -7931 8220 8625 -8032 -9096 343 -7601 218 -5790 -3514 789 6867 41 1413 7651 8767 1817 345 -5305 6981 -2250 6463 -1637 -3336 3657 -33 -4254 -7809 -1635 -4989 -6279 3572 9210 -9052 2386 2610 -4273 -9195 6958 5627 2417 23 5818 6652 6560 -3311 5710 -2701 -2295 6211 -9726 9966 -1146 -6907 -6286 3274 5434 972 8594 -2665 -970 7883 -3813 3976 2196 -5099 2666 7132 6910 -8437 2553 438 1195 -1609 -539 -9200 -8530 -4743 6568 6411 9201 4788 -7292 -7879 -2294 -5970 4798 -8102 8192 -7104 -2823 -5027 -9277 -7700 -577 1998 8783 7479 9462 951 -9421 6484 -326 -891 -4532 -3004 -5114 7008 5652 7130 -5442 -1064 8806 -8242 2327 1221 -2218 -1241 -8082 5210 -915 610 -114 8553 3218 6572 -7509 -2992 1447 -8651 -3676 1592 -2143 -2940 4828 4221 4623 -107 3680 -5060 -3068 7958 -9753 -7649 8009 -9637 2737 -2836 -525 -8780 8838 3429 -4377 -6979 -3348 7622 4303 8262 2166 7614 -412 4776 -3624 1650 53 2157 6696 -935 7341 338 -5246 4416 8034 -3946 6903 -9525 -4905 -2668 6412 -7353 -7691 -8612 5510 -3280 9087 -8337 -1104 -6157 -5263 6396 -3900 9150 6616 -756 -3609 8772 -1228 4578 5613 7773 8754 6703 -580 -1027 -9047 9630 3025 5277 -6218 2179 2791 1736 1165 1175 4001 -8898 6005 3321 8166 5190 -760 7518 -9175 -1550 -623 -67 -4689 6961 -874 9772 -4184 -9843 4107 1571 7422 -7771 7623 -3479 6991 266 -3009 2108 -9773 -3485 -8499 8371 1586 -1053 -9300 3892 3333 -6476 -8076 -4333 5213 -3662 -1920 -7159 1162 6345 -1845 -4139 -8895 6413 -730 -5623 -2275 -542 -1953 -1655 259 -3237 457 4173 -2494 1440 -2299 5522 7770 4289 3726 -1296 -2989 1713 -2369 1347 -8693 945 -7907 7848 3901 -8901 157 615 -231 -9103 -5632 7854 -2141 2245 -7005 2881 1921 -6263 845 -9663 -4802 5219 -8527 -2856 -2709 -2855 3586 2539 3498 -9189 6761 7702 6683 8171 5730 6795 5070 -6559 -4835 -5958 -7603 -8796 7147 9794 -8960 9180 843 -6038 117 -5535 -1802 2604 -8841 -1017 2353 -1193 8195 -740 2377 -5628 5322 -851 -8892 2788 -1243 -7687 5512 1433 8817 6024 -2283 6747 3152 3340 471 4444 7454 8692 518 3869 -2019 -545 -8425 -8219 5998 271 -7873 9135 4911 -3457 -4385 -890 2473 5895 -681 -7370 -441 -2210 -8614 -4250 967 9131 -9278 -1958 1324 -9613 -8094 -1218 -5496 -5160 -6065 822 -2585 6084 -976 1935 2837 1693 -4544 -552 2678 -8984 7883 5354 5363 -4440 9783 4907 -5589 4402 -3150 -5335 -8546 -7804 1261 9871 -7443 2076 -6905 -4638 1423 -2597 -1291 -4980 3194 -7495 2797 -3923 4775 -6181 -6323 2128 -3917 2370 -325 7757 -1485 -2249 -6926 4001 7906 8088 6560 9094 -1233 -7812 -8661 -9645 -2905 -3710 -205 -8492 9242 -1228 4425 -781 -9910 -9979 4501 7236 -5180 -5126 -902 -673 337 7689 -3797 8660 