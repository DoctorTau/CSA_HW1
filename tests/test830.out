-7838 -2856 -7054 -2638 -8564 -6858 -7740 -4696 -8662 -9948 -5564 -6722 -5184 -6590 -8826 -6532 -6748 -1414 -1694 -2664 -2216 -8078 -6580 -3530 -2384 -770 -8032 -9096 -5790 -3514 -2250 -3336 -4254 -9052 -9726 -1146 -6286 -970 -9200 -8530 -7292 -2294 -5970 -8102 -7104 -7700 -326 -4532 -3004 -5114 -5442 -1064 -8242 -2218 -8082 -114 -2992 -3676 -2940 -5060 -3068 -2836 -8780 -3348 -412 -3624 -5246 -3946 -2668 -8612 -3280 -1104 -3900 -756 -1228 -580 -6218 -8898 -760 -1550 -874 -4184 -9300 -6476 -8076 -3662 -1920 -730 -542 -2494 -1296 -5632 -4802 -2856 -5958 -8796 -8960 -6038 -1802 -740 -5628 -8892 -890 -7370 -2210 -8614 -4250 -9278 -1958 -8094 -1218 -5496 -5160 -976 -4544 -552 -8984 -4440 -3150 -8546 -7804 -4638 -4980 -6926 -7812 -3710 -8492 -1228 -9910 -5180 -5126 -902 